src/main/java/io/aiven/kafka/connect/opensearch/BulkProcessor.java

```diff
@@ -110,6 +125,18 @@ public BulkProcessor(final Time time,
                 BehaviorOnMalformedDoc.IGNORE,
                 BehaviorOnMalformedDoc.WARN);
         }
+
+        if (reporter != null && this.errorToleranceType == ErrorToleranceType.NONE
+                && (config.behaviorOnMalformedDoc() != BehaviorOnMalformedDoc.FAIL
+                || config.behaviorOnVersionConflict() != BehaviorOnVersionConflict.FAIL)) {
+            LOGGER.warn("The '{}' is set to `{}`, which means that only errors that would cause the task to fail"
+                        + " will be reported to the dead letter queue. You may consider setting property '{}' to '{}'"
+                        + " to deal with errors that are configured to be ignored or warned",
+                    OpensearchSinkConnectorConfig.ERRORS_TOLERANCE_CONFIG,
+                    ErrorToleranceType.NONE,
+                    OpensearchSinkConnectorConfig.ERRORS_TOLERANCE_CONFIG,
+                    ErrorToleranceType.ALL);
+        }
```

Comment:
Is this still accurate? Don't we report errors (which does not necessarily mean send them to a DLQ topic!) if the behavior on malformed docs or on version conflicts is set to `REPORT` now?

I'm not sure we need any warning messages here--for all of the permutations I can think of for properties related to errors tolerance, behavior on malformed docs/version conflicts, etc., there's at least one circumstance in which each would make sense.